values/src/context.rs

```diff
@@ -97,14 +96,17 @@ impl ParsingContext<Dimension> for ResolverContext {
                                 },
                             }
                         } else if let Some(end) = interval.end {
+                            if datetime_value.datetime_kind == DatetimeKind::Date || datetime_value.datetime_kind == DatetimeKind::Time {
+                                eprintln!("Warning: {:?} kind with an interval - {:?}", datetime_value.datetime_kind, interval);
+                            }
```

Comment:
In order to properly log a warning, you should use the `log` crate:

```toml
[dependencies]
log = "0.4"
```

Then import the `warn!()` macro: `use log::warn;` and use it like `eprintln!`:
```rust
warn!("{:?} kind with an interval - {:?}", datetime_value.datetime_kind, interval);
```

(No need to prefix with "Warning" as it will be included in the log output)